src/Deposit.js

```diff
@@ -224,8 +243,8 @@ export class DepositFactory {
       await this.systemContract.methods.getNewDepositFeeEstimate().call()
     )
 
-    const accountBalance = await this.config.web3.eth.getBalance(
-      this.config.web3.eth.defaultAccount
+    const accountBalance = toBN(
+      await this.config.web3.eth.getBalance(this.config.web3.eth.defaultAccount)
```

Comment:
This is filed under “that didn't even work, but it failed in a way that allowed things to continue as expected”.

Reply:
Running the local test, this new change is causing an error for the next line:
```
{
    "type": "error",
    "timestamp": "Tue, 18 Aug 2020 22:35:16 GMT",
    "arguments": {
      "0": {
        "message": "Insufficient balance 1000000000000000000000000000000124273889123276000000000 to open deposit (required: 1164000000000000).",
        "originalStack": [
          "Error: Insufficient balance 1000000000000000000000000000000124273889123276000000000 to open deposit (required: 1164000000000000).",
          "at DepositFactory.createNewDepositContract (http://localhost:3000/static/js/main.chunk.js:1082:13)",
          "at async Function.forLotSize (http://localhost:3000/static/js/main.chunk.js:1124:9)"
        ],
        "error": {}
      }
    },
```

This happens when I select the lot size of 0.01 BTC

Reply:
Oh goodness.

Yeah that check is inverted 😂/:sob: Will change it to `.gte`.

Reply:
I think maybe the check itself is okay - `creationCost.lt(accountBalance)` but I think the param for `.lt` needs to be a plain JS number instead of a BN?